client/my-sites/checkout/checkout-thank-you/index.tsx

```diff
@@ -913,10 +897,9 @@ export class CheckoutThankYou extends Component<
 					primaryCta={ this.primaryCta }
 					displayMode={ displayMode }
 					purchases={ purchases }
-					isRedesignV2={ isRedesignV2( this.props ) }
 					currency={ receipt.data?.currency }
 				>
-					{ ! isRedesignV2( this.props ) && ! isSimplified && primaryPurchase && (
```

Comment:
Similarly, we no longer need the redesign check here. All redesigned pages should render using the logic defined by these lines: https://github.com/Automattic/wp-calypso/pull/86271/files#diff-daac408bb350333e9fb8ca7c03233e75f03656e02a45c152e574227e6d253ee7R569-R610